src/components/Clusters/views/ClusterOverview/ClusterDetails.js

```diff
@@ -97,6 +103,13 @@ export default function ClusterDetails({ currentCluster }) {
                 </DynamicPageComponent.Column>
               </>
             )}
+            {!environmentParametersLoading && (
```

Comment:
we don't want to show it at all if there is no IPs available
<img width="502" height="257" alt="obraz" src="https://github.com/user-attachments/assets/a082d3cc-9a87-47a5-b74e-dec32d31c3e4" />